docs/specs/resolve.yml

```
@@ -265,6 +265,22 @@ outputs:
   .errors.log: |
     {"message_severity":"error","code":"include-not-found","message":"Invalid include link: 'redirect.md'.","file":"docs/a.md","line":1,"column":9}
 ---
+# Show warning if inclusion is using links not `.md` extension
+dryRunOnly: true
+inputs:
+  docfx.yml:
+  docs/a.md: |
+    [!include[B](../media/b.png)]
+    Link to [!include[C](../c.ps1)]
+    Empty inclusion [!include[D]()]
+  media/b.png: Token
+  c.ps1: Token
+outputs:
+  .errors.log: |
+    {"message_severity":"warning","code":"include-invalid","message":"Invalid include link extension: '../media/b.png'.","file":"docs/a.md","line":1,"column":1}
```

@yufeih Would it be better if the message changed to the following?

```suggestion
    {"message_severity":"warning","code":"include-invalid","message":"Invalid include link extension: '.png', the '!include' syntax only supports '.md' extension.","file":"docs/a.md","line":1,"column":1}
```